Undefined reference error to uv__time_precise() with gcc 5

On an Ubuntu 15.10 (x86_64/Linux) with gcc 5.2.1, I get the following errors when building libtuv with `make`:

```
In file included from /home/akiss/devel/libtuv/include/uv.h:54:0,
                 from /home/akiss/devel/libtuv/source/linux/uv_linux_thread.c:40:
/home/akiss/devel/libtuv/source/linux/uv_platform.h:102:17: warning: inline function ‘uv__time_precise’ declared but never defined
 inline uint64_t uv__time_precise();

In file included from /home/akiss/devel/libtuv/include/uv.h:54:0,
                 from /home/akiss/devel/libtuv/test/runner_linux.c:44:
/home/akiss/devel/libtuv/source/linux/uv_platform.h:102:17: warning: inline function ‘uv__time_precise’ declared but never defined
 inline uint64_t uv__time_precise();

Linking C executable ../bin/tuvtester
../lib/libtuv.a(uv_unix_thread.c.o): In function `uv_cond_timedwait':
/home/akiss/devel/libtuv/source/unix/uv_unix_thread.c:311: undefined reference to `uv__time_precise'
```
(The warnings repeat several times.)

--- a/source/linux/uv_platform.h
+++ b/source/linux/uv_platform.h
@@ -99,7 +99,7 @@ uint64_t uv__hrtime(uv_clocktype_t type);
 #define uv__update_time(loop)                                                 \
   loop->time = uv__hrtime(UV_CLOCK_FAST) / 1000000
 
-inline uint64_t uv__time_precise();
+uint64_t uv__time_precise();
 
 
 //-----------------------------------------------------------------------------

--- a/source/mbed/uv_platform.h
+++ b/source/mbed/uv_platform.h
@@ -78,7 +78,7 @@ uint64_t uv__hrtime(uv_clocktype_t type);
 #define uv__update_time(loop)                                                 \
   loop->time = uv__hrtime(UV_CLOCK_FAST) / 1000000
 
-inline uint64_t uv__time_precise();
+uint64_t uv__time_precise();
 
 void tuv__time_init(void);
 

--- a/source/nuttx/uv_platform.h
+++ b/source/nuttx/uv_platform.h
@@ -106,7 +106,7 @@ uint64_t uv__hrtime();
 #define uv__update_time(loop)                                                 \
   loop->time = uv__hrtime() / 1000000
 
-inline uint64_t uv__time_precise();
+uint64_t uv__time_precise();
 
 
 //-----------------------------------------------------------------------------

